### DISCUSSION TITLE
Invalid plot file format

### DISCUSSION THREAD

**Discussion by sbc-juangomez** [Chia-Network/chia-blockchain#16429] [Support]
Hello,

Cultivate a few months of plots compression, I use Chia 2.0.1 version and to load compressed plots I see this error:

`2023-09-19T21:40:57.635 harvester chia.plotting.manager   : ERROR    Failed to open file D:\plots\plot-k32-c8-2023-08-07-13-58-6a7b5782f8a3a316b4e2b35ac0401fd880354e48d24107119117bb59fade43d8.plot. Invalid plot file format Traceback (most recent call last):
  File "chia\plotting\manager.py", line 326, in process_file
ValueError: Invalid plot file format`

pc spects:
Windows 10:
AMD Ryzen 7 370x
64gb ram ddr4
NVIDIA RTX 3070

**Comment by BrandtH22** [comment]
Hey @sbc-juangomez , there is a known bug for plotting on windows (especially win 10) that leads to ~3% bad plots.
We recommend using the `chia plots check` command to scan the plots and then manually remove any that are failing:
https://docs.chia.net/cli#check

Note - we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
